changelogs/8.18.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[apm-release-notes-8.18]]
```

Comment:
@carsonip , same question, can I keep it or i need to delete it?

Reply:
deleting it from this backport PR is fine